lib/cassette.rb

```diff
@@ -35,6 +37,12 @@ def logger
 
   def config
     @config if defined?(@config)
+
+    @config.tls_version = DEFAULT_TLS_VERSION if @config.tls_version.nil?
+
+    @config.verify_ssl = DEFAULT_VERIFY_SSL if @config.verify_ssl.nil?
```

Comment:
What about changing the README to show that it is possible to set these configs `tls_version` and `verify_ssl' ?

Reply:
@fabioperrella 

Nice! I made the change and sent it to the PR, can you check again please?